src/lib.rs

```diff
@@ -9,6 +9,7 @@
 
 #[cfg(feature = "alloc")]
 extern crate alloc;
+extern crate core;
```

Comment:
why do we need this?

https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html#an-exception
> [core](https://doc.rust-lang.org/core/index.html): Usually this is not necessary, because core is automatically imported, unless the crate is marked with [#![no_core]](https://github.com/rust-lang/rust/issues/29639). For example, some of the internal crates used by the standard library itself need this.